src/components/ContentSidebar/MetadataSidebar.js

```diff
@@ -313,17 +315,27 @@ class MetadataSidebar extends React.PureComponent<Props, State> {
      * @return {void}
      */
     fetchFileErrorCallback = (e: ElementsXhrError, code: string) => {
-        this.commonErrorCallback(e, code, { error: messages.sidebarFileFetchingErrorContent, file: undefined });
+        this.onApiError(e, code, { error: messages.sidebarFileFetchingErrorContent, file: undefined });
     };
 
     /**
-     * Handles a successful file fetch
+     * Handles a successful file fetch.
+     * Can be called multiple times when refreshing caches.
+     * On file load we should fetch metadata, but we shouldn't need to fetch
+     * if the file permissions haven't changed from a prior file fetch.
+     * Metadata editors mostly care about upload permission.
      *
      * @param {Object} file - the box file
      * @return {void}
      */
     fetchFileSuccessCallback = (file: BoxItem) => {
-        this.setState({ file }, this.fetchMetadata);
+        const uploadPermission = 'permissions.can_upload';
```

Comment:
I'll de-dupe this string also in another PR, make it a global constant, being repeated in other areas too